nilearn/plotting/displays.py

```diff
@@ -870,8 +945,13 @@ def annotate(self, left_right=True, positions=True, size=12, **kwargs):
         positions: boolean, optional
             If positions is True, annotations indicating the
             positions of the cuts are drawn.
+        scalebar: boolean, optional
+            If scalebar is True, cuts are annotated with a reference
+            scale bar (bottom-right position by default).
         size: integer, optional
             The size of the text used.
+        scalebar_kwargs: dict, optional
+            Arguments to control how the scale bar is drawn.
```

Comment:
I would remove these and point to the method "draw_scale_bar" if people want finer control.

Reply:
One option is to extract the most import options and add them here explicitly, eg "scalebar_units", "scalebar_pos". The reason that I am insisting is that "kwargs" are really not user friendly for non experts.